packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/EditView/index.js

```diff
@@ -40,6 +43,10 @@ import { data as permissions } from './utils/tests/dataMock';
 import init from './init';
 import reducer, { initialState } from './reducer';
 
+const ButtonWithRightMargin = styled(Button)`
+  margin-right: 8px;
```

Comment:
Could you please use a theme spacing value instead? https://design-system-git-main-strapijs.vercel.app/?path=/story/design-system-components-theme--spaces

Reply:
ok

Reply:
done